concepts/teams-changenotifications-teammembership.md

```diff
@@ -0,0 +1,155 @@
+---
+title: "Get change notifications for any change in Teams membership using Microsoft Graph"
+description: "Get change notifications for any change in Teams membership using Microsoft Graph"
+author: "Abhishek Anand"
+localization_priority: Priority
+ms.prod: "microsoft-teams"
+ms.custom: scenarios:getting-started
+---
+
+# Get change notifications for change in teams membership using Microsoft Graph
+
+Change notifications enable you to subscribe to changes (create, update, and delete) to [messages](/graph/api/resources/chatMessage?preserve-view=true) in a [channel](/graph/api/resources/channel?preserve-view=true) or [chat](/graph/api/resources/chat?preserve-view=true). Change notifications provide a low latency model by allowing you to maintain a [subscription](/graph/api/resources/webhooks?preserve-view=true). You can also get the resource data in the notifications and therefore avoid calling the API to get the payload.
```

Comment:
Update

Reply:
In case of membership subscription,
create means Add member,
delete means Remove member,
update means Member update, which is essentially 'Role update' as of today